src/main/java/com/rybka/config/PropertyInfo.java

```diff
@@ -5,4 +5,10 @@
     public static final String PROPERTY_EXCHANGE_SOURCE = "exchange.source";
     public static final String PROPERTY_EXCHANGE_KEY = "exchange";
     public static final String PROPERTY_PRIME_EXCHANGE_KEY = "prime_exchange";
+
+    public static final String PROPERTY_EXPORT_FOLDER = "export.folder";
+    public static final String PROPERTY_EXPORT_TYPE = "export.type";
+    public static final String PROPERTY_EXPORT_CONSOLE = "console";
```

Comment:
This isn't export property, it's values.

Reply:
meant for console, csv, json

Reply:
Don't think we should keep that values in properties files. 
Let's create an Enum for export values.

Reply:
fixed